zetasql-toolkit-bigquery/src/main/java/com/google/zetasql/toolkit/catalog/bigquery/BigQueryCatalog.java

```diff
@@ -297,6 +297,23 @@ private List<String> buildCatalogNamesForResource(String reference) {
     }
   }
 
+  private List<String> buildCatalogNamesForWildcardTable(String wildcardTableReference) {
```

Comment:
If we make `BigQueryReference` support resources ending in `*`, we wouldn't need this method anymore. The existing `buildCatalogNamesForResource()` has this exact behaviour.